helsingborg-stad/app-mitt-helsingborg
# Feat/sentry monitoring

## Explain the changes you’ve made
Implemented Sentry monitoring for the application in production environment

## Explain why these changes are made
We want to track errors that occurs in the application in our production environment

## Explain your solution
Created a factory which returns a sentry service while in production. This service will then communicate with our provided sentry server which is going to be added as an environment variable

## How to test

1. Checkout this branch
2. Fire upp the simulator by running the command `yarn ios` or `yarn android`
3. In development it should run as usual
4. If testing app in production (APP_ENV = production), then ask @kajm for credentials

## Tested environments

- [] Storybook on a iOS device/simulator.
